src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --white: #fff;
+  --blue: #00acdc;
+  --navy-blue: #060b35;
+}
+
+.card {
```

Comment:
Also, move it in `card.css` file and then import all files `index.css` file (or `style.css`)